app/assets/stylesheets/header.scss

```diff
@@ -0,0 +1,67 @@
+@import "themeColors";
+
+#logo {
```

Comment:
This css was removed and is not used anymore. Was the PR rebased with master? This is not present currently in master. Don't know if other css that was removed was added again

Reply:
The #logo part

Reply:
Removed #logo. Checked the rest of the rules and they exist on master.